libcodechecker/libhandlers/store.py

```diff
@@ -107,6 +107,13 @@ def add_arguments_to_parser(parser):
                              "the '--name' parameter given to 'codechecker-"
                              "analyze' will be used, if exists.")
 
+    parser.add_argument('--tag',
+                        type=str,
+                        dest="tag",
+                        required=False,
+                        default=argparse.SUPPRESS,
+                        help="Tag which identifies a certain run update.")
```

Comment:
Okay... there are a lot of concepts not really explained here. "Update"? "Tag"? I think this help message should be made better...

I believe `An identifier for this individual store of results in the run's history.` (Run history is used on the GUI.)